app/assets/stylesheets/questions.css.scss

```diff
@@ -29,6 +33,10 @@
   margin-top: 0px;
 }
 
+.qset-name-modal {
```

Comment:
`.btn-generate-q` looks to be ordered alphabetically, can we do that with thise as well?